longevity_test.py

```diff
@@ -78,6 +95,21 @@ def run_pre_create_keyspace(self):
         if self.params.get('pre_create_keyspace'):
             self._pre_create_keyspace()
 
+    def _run_validate_large_collections_in_system(self, node):
+        self.log.info("Verifying large collections in system tables on node: {}".format(node))
+        with self.db_cluster.cql_connection_exclusive(node=node) as session:
+            query = "SELECT * from system.large_cells WHERE keyspace_name='large_collection_test'" \
+                    " AND table_name='table_with_large_collection' ALLOW FILTERING"
+            statement = SimpleStatement(query, fetch_size=10)
+            data = list(session.execute(statement))
+            assert len(data) >= 1, "Did not find expected row in system.large_cells"
```

Comment:
Not sure if "assert" is the correct behaviour here, maybe "Error" event is better.

Reply:
Sorry for stupid question, but do you mean just to use `if` statement and print LOGGER.ERROR message ?
This is the main idea behind this test to check the table has an entry for large collections and relevant message appear in the logs.

Reply:
This is not the main idea being this test, it's another validation you added to this test scenario which tests how scylla behaves with large collections.

So, error event is better IMO.

Reply:
Ok, then I change this to `if` and `self.log.error`.

Reply:
`self.log.error` is not the same as an `error event` FYI

Reply:
Changed to InfoEvent("message", severity=Severity.ERROR)